src/gui/editors/AutomationEditor.cpp

```diff
@@ -1432,13 +1143,12 @@ void AutomationEditor::paintEvent(QPaintEvent * pe )
 				++it;
 			}
 
-			for( int i = it.key(), x = xCoordOfTick( i ); x <= width();
-							i++, x = xCoordOfTick( i ) )
+			for (int i = POS(it), x = xCoordOfTick(i); x <= width();
+							i++, x = xCoordOfTick(i))
```

Comment:
If it's too long a line, should we format like this?
```suggestion
			for (
				int i = POS(it), x = xCoordOfTick(i);
				x <= width();
				i++, x = xCoordOfTick(i)
			)
```
If we change the coding conventions to allow braces on the same line (discussed briefly before) we could end up with:
```suggestion
			for (
				int i = POS(it), x = xCoordOfTick(i);
				x <= width();
				i++, x = xCoordOfTick(i)
			) {
```
which would save some whitespace, too.